Source/JavaScriptCore/yarr/YarrInterpreter.cpp

```diff
@@ -115,9 +115,9 @@ class Interpreter {
 
         static size_t allocationSize(unsigned numberOfFrames)
         {
-            static_assert(alignof(DisjunctionContext) <= sizeof(void*));
+            static_assert(alignof(DisjunctionContext) <= alignof(max_align_t));
```

Comment:
Maybe we should leave a comment here about the reasoning. I.e, this is the guaranteed alignment from malloc